Header header

float32[3] global_vel
float32[3] global_accel_avg
